edk2toollib/uefi/wincert.py

```diff
@@ -52,41 +57,64 @@ class WinCertPkcs1(object):
 
     def __init__(self, filestream=None):
         """Inits the object."""
-        if (filestream is None):
+        if filestream is None:
             self.Hdr_dwLength = WinCertPkcs1.STATIC_STRUCT_SIZE
             self.Hdr_wRevision = WinCert.REVISION
             self.Hdr_wCertificateType = WinCert.WIN_CERT_TYPE_EFI_PKCS115
-            self.HashAlgorithm = None
```

Comment:
same as the previous. We either need to using getter / setter to throw warnings for these values, or I can do a minor level release and tell everyone we have switched from CamelCase to SnakeCase for these class attributes and that they should do the methods too, but that for now those are only deprecated